configuration/env_var_processors.rst

```diff
@@ -190,6 +190,12 @@ Symfony provides the following env var processors:
 ``env(float:FOO)``
     Casts ``FOO`` to a float.
 
+``env(date_time:FOO)``
```

Comment:
event if other example uses `FOO`, what about here use a real date_time value format ?
I mean is it the Y-m-d H:i:s ?
obviously, for ``env(int:FOO)`` we know int refers to what, but date time is more ambiguous and doc must be straightfroward 
wdyt?

Reply:
Documentation updated to the latest implementation. I have added the "version added directives" and the usage examples to describe how to use the both processors. I think that the examples are sufficient and the `FOO` can stay in processor name to make the documentation consistent.